README.md

```diff
@@ -38,11 +44,12 @@ This action can be used to redeploy application on the DigitalOcean's [App Platf
                       ]'
   ```
 - DigitalOcean App Platform will now update your DOCR information in App Spec and then deploy your application.
-  (Please use unique tag value for DigitalOcean Container Registry Push instead of latest to update DOCR update in App Spec)
+
+**Note: Always use unique tag names to push image to the DigitalOcean Container Registry. This will allow you to deploy your application without delay. [ref](https://docs.digitalocean.com/products/container-registry/quickstart/)**
 
 # Inputs
 - `app_name` - Name of the app on App Platform.
-- `list_of_image` - (optional)List of json object for providing information about name,repository and tag of the image in docr.(By default tag of the image is latest)
+- `list_of_image` - (optional)List of json object for providing information about name, repository and tag of the image in docr.(by default latest tag is used)
```

Comment:
can we rename this to `images`?

Reply:
also we advise against using `latest` should we use the commit short sha by default?

Reply:
I believe user should have option to declare their own tags. For example; they may prefer to tag images by version number which might be unique.
Saying that we can provide an optional feature to deploy using sha tag but as users are deploying their docker images to docr by themselves we might need to provide handling of deployment from our side. For example; We take same input except tag field and then deploy dockerfile to docr with generated sha tag and then update our app platform app spec.

Reply:
I'm suggesting we change the default from `latest` which has known issues to something that doesn't and is common across all projects. They're still welcome to set whatever they want but we can give them a nice default that is going to work without issue.

Reply:
Actually they are the one pushing the images and we are the one pulling so what i am trying to say is they need to push image using sha hash. The default is latest because if the user does not mention tag DOCR uses latest as tag. If we set sha hash as default it might confuse users as now they have to explicitly set sha key and also push image using same key. This might be good in regulating the flow but I believe there may be many users who are not using tags for docr upload. So for example; I am user writing github action and I forgot to tag the image our action will be trying to retrieve tag based on hash rather then using latest.

Reply:
Aaah I see, you're right. I misunderstood! thanks for explaining 👍